core/system.go

```diff
@@ -654,6 +654,17 @@ func (s *ABSystem) RunOperation(operation ABSystemOperation) error {
 			PrintVerbose("ABSystemRunOperation:err(4.0.1): %s", err)
 			return err
 		}
+	} else {
+		err := os.RemoveAll(systemNew)
+		if err != nil {
+			PrintVerbose("ABSystemRunOperation:err(4.0.2): could not cleanup old systemNew folder %s", err)
+			return err
+		}
+		err = exec.Command("cp", "--reflink", "-a", systemOld, systemNew).Run()
+		if err != nil {
+			// can be safely ignored
```

Comment:
A warning should be shown to the user if cp fails to inform that a slower method of copying will be used.

Reply:
An error could occur for many reasons other than the filesystem not supporting CoW, it would be better to add a general warning like `reflink copy of system failed, falling back to slow copy`

Reply:
ok, done